roles/ansible/tower/config-ansible-tower-ocp/README.md

```diff
@@ -6,45 +6,42 @@ This role is used to deploy and configure an Ansible Tower running as containers
 ## Requirements
 
   - A running OpenShift Cluster and installed 'oc' client in the Ansible host
-  - An existing project where to deploy Ansible Tower (must match the `ansible_tower.openshift_project` variable)
-  - An existing PVC for Postgresql named `postgresql-data`
 
 
 ## Role Variables
 
-The variables used to install an Ansible Tower instance are outlined in the table below.
+The variables used to install an Ansible Tower instance are outlined in the table below. Authentication shall be based on either username/token, with token being prefered. You can specify username and password but role will automatically retrieve token.
 
 | Variable | Description | Required | Defaults |
 |:---------|:------------|:---------|:---------|
-|ansible_tower.openshift_host|OpenShift API url|yes||
-|ansible_tower.openshift_project|Project where to deploy Tower|yes|'ansible-tower'|
-|ansible_tower.openshift_user|User to login into openshift|yes||
-|ansible_tower.openshift_password|Openshift user password|yes||
-|ansible_tower.admin_password|Tower admin user password|yes||
-|ansible_tower.secret_key|Openshift token information|yes||
-|ansible_tower.pg_username|Postgresql User|yes||
-|ansible_tower.pg_password|Postgresql Password|yes||
-|ansible_tower.rabbitmq_password|RabbitMQ password to use|yes||
-|ansible_tower.rabbitmq_erlang_cookie|RabbitMQ cookie|yes||
+|openshift_host|OpenShift API url|yes|CRC on local host|
+|openshift_project|Project where to deploy Tower|yes|'tower'|
+|openshift_user|User to login into openshift|yes|"kubeadmin"|
+|openshift_password|Openshift user password|yes(either that or token)||
+|openshift_token|Openshift token|yes(either that or password)||
+|admin_user|Tower admin username|yes|"admin"|
+|admin_password|Tower admin user password|yes|"admin"|
+|secret_key|Tower secret key|yes|"abcdefgh"|
 
```

Comment:
Quite a few entries missing here - e.g.: `pg_*`,`openshift_pg_*`, and a few others. Please update per your `defaults` file and otherwise what the tasks and template calls for.